hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -2539,4 +2539,9 @@ default BalanceResponse balanceRSGroup(String groupName) throws IOException {
    */
   List<LogEntry> getLogEntries(Set<ServerName> serverNames, String logType, ServerType serverType,
     int limit, Map<String, Object> filterParams) throws IOException;
+
+  /**
+   * Force request flush master local region
+   */
+  void forceRequestFlushMasterStore() throws IOException;
```

Comment:
I mean add a force flag... Not add a force to the method name...

Checked the code, we do not have this flag in the Admin interface, and at rs side, it will always use true when calling region.flush...

So sorry, let's keep align with other methods in this interface, just call it flushMasterStore, and at master side, we call region.flush(true).

Reply:
If we call region.flush(true) at the master side, the changesAfterLastFlush value of MasterRegionFlusherAndCompactor will become incorrect, which may trigger an invalid flush operation.
But we can call MasterRegionFlusherAndCompactor.requestFlush() to avoid this.

Do we need to consider this situation?

Reply:
Then we could introduce a method in MasterRegionFlusherAndCompactor? Just reset changesAfterLastFlush after calling flush?

Anyway, what I mean is we should try our best to align the behavior in our Admin interface.

Reply:
Yes, we need to introduce a new method to reset the value of changesAfterLastFlush after calling flush.
Let me modify the code here. Thanks @Apache9 .

Reply:
I checked the code, because region.flush() is a synchronous method, we should call resetChangesAfterLastFlush() before calling region.flush() method, which should be similar to MasterRegionFlusherAndCompactor.flushLoop() method code logic.  @Apache9